lib/snmp.js

```diff
@@ -759,8 +760,8 @@ Session.prototype.getSubtree = function (options, callback) {
             if (inTree(options.startOid, varbinds[0].oid)) {
                 if (varbinds[0].value === 'endOfMibView' || varbinds[0].value === 'noSuchObject' || varbinds[0].value === 'noSuchInstance') {
                     callback(null, vbs);
-                } else if (vbs.length && vbs.slice(-1)[0].oid.slice(-1)[0] >= varbinds[0].oid.slice(-1)[0]) {
-                    return callback(new Error('OID not increasing'))
+                } else if (vbs.length && compareOids(vbs.slice(-1)[0].oid, varbinds[0].oid) === -1) {
```

Comment:
If compareOids() returns 0 (ie. the oids are equal) the oid is also not increasing. Perhaps make this comparison 
`compareOids() <== 0`